leetcomp/parse.py

```diff
@@ -83,9 +83,18 @@ def parsed_content_is_valid(parsed_content: list[dict[Any, Any]]) -> bool:
     return True  # Parsed content is valid if no assertions fail
 
 
+def extract_interview_exp(content: str) -> str:
+    match = re.search(
+        r"https://leetcode.com/discuss/interview-experience/\S+", content
```

Comment:
keep this pattern at the top of the file after the imports, interview_exp_pattern = re.compile("...")